src/schemas/Clinical/gqlTypeDefs.ts

```diff
@@ -167,6 +167,64 @@ export default gql`
     donorId: String!
   }
 
+  """
+  Clinical Data Schemas
+  """
+  input ClinicalInput {
```

Comment:
The comment text like used here gets added to the GQL Schema and is displayed in the gql docs that users see when using the API. Can you add some more of these descriptions to the other types you have added? Since many of these have similar names, some description of the differences will likely help make sense of it for future devs.